### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by aneallaryea100**
## Blog app - creating a data model
- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- Build your project schema.
- Create and run the necessary migration files.
- Table and column names should match the ERD diagram.
- note: photo for users table should be a link to an image
- Foreign keys should be included.
- All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20221026150443_add_users_to_likes.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUsersToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
```
Comment (omar-muhamad) [inline db/migrate/20221026150443_add_users_to_likes.rb]: - [ ] Please add the foreign key between likes and users tables as `author_id`, not `user_id` as mentioned on the ERD table.

image

[Hint] You can do it like that:
```ruby
add_reference :likes, :author, foreign_key: { to_table: :users }, null: false
```

File: db/migrate/20221026151111_add_users_to_comments.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUsersToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
```
Comment (omar-muhamad) [inline db/migrate/20221026151111_add_users_to_comments.rb]: - [ ] Same here, Please add the foreign key between comments and users tables as `author_id`, not `user_id` as mentioned on the ERD table.

File: db/migrate/20221026151206_add_users_to_posts.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUsersToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
+end
```
Comment (omar-muhamad) [inline db/migrate/20221026151206_add_users_to_posts.rb]: - [ ] Same here, Please add the foreign key between posts and users tables as `author_id`, not `user_id` as mentioned on the ERD table.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @aneallaryea100,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @aneallaryea100👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**